manifests/overlays/service-mesh/OWNERS

```diff
@@ -0,0 +1,10 @@
+# Each list is sorted alphabetically, additions should maintain that order
```

Comment:
why do we have this OWNERS files in the manifest, I don't understand

Reply:
OWNERS files can be placed at lower levels to support "just these files" changing. So if something under the `overlays/service-mesh` folder is the only thing to change, they would need need the repo ownership to step in to review it. It's additive on 

I think this is fine for this use-case... We should definitely avoid OWNERS files over code so that we can still approve the holistic nature of that.

Reply:
> why do we have this OWNERS files in the manifest, I don't understand

Our line of thinking is that we offer help and knowledge to keep this thing up-to-date and evolving. So if anything will be proposed to change here we will be pinged